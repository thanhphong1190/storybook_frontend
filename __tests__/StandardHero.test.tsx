import {
  StandardHero,
  StandardHeroProps
} from '../src/components/standard-hero/StandardHero'
import renderer from 'react-test-renderer'

const props: StandardHeroProps = {
  imgURL: '/example/home-hero.png',
  title: 'Promotion'
}

it('StandardHero renders correctly', () => {
  const tree = renderer.create(<StandardHero {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
