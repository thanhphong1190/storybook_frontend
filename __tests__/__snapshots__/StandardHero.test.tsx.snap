// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StandardHero renders correctly 1`] = `
<section
  className="StandardHerostyles__Section-sc-173nurp-0 cvViDL"
>
  <div
    className="StandardHerostyles__Container-sc-173nurp-1 heWpyi"
  >
    <div
      className="StandardHerostyles__TitleWrapper-sc-173nurp-2 kFyHRR"
    >
      <div
        className="globalstyles__Text-sc-1nc69wi-9 bpjwgw"
        color="blue-dark"
        fontWeight="bold"
        size="4xl"
      >
        Promotion
      </div>
    </div>
    <img
      className="StandardHerostyles__IMG-sc-173nurp-3 gjoPts"
      src="/example/home-hero.png"
    />
  </div>
</section>
`;
